setup.cfg

```diff
@@ -1,12 +1,102 @@
+[metadata]
+name = ctapipe
+description = CTA Python pipeline experimental version
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = ctapipe developers
+author_email = karl.kosack@cea.fr
+license = BSD 3-Clause License
+url = https://github.com/cta-observatory/ctapipe
+classfisers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Astronomy
+    Development Status :: 3 - Alpha
+
+
+[options]
+packages = find:
+python_requires = >=3.8
+zip_safe = False
+install_requires=
+	astropy~=5.0
+	bokeh~=2.0
+	docutils
+	eventio>=1.9.1<2.0.0a0
+	h5py
```

Comment:
Do we still need h5py now that we do our own table writing and reading? I think it was only required for astropy's HDF5 output.

Reply:
h5py is required to support reading tables written using the astropy mechanism using `ctapipe.io.read_table`. We have a unit test for that and I think it's a nice optional feature.

So I moved h5py from the required dependencies to the optional `tests` and `all` extras and left it in the environment.